client/my-sites/checkout/controller.jsx

```diff
@@ -22,7 +22,7 @@ import { isGSuiteRestricted } from 'lib/gsuite';
 import { getRememberedCoupon } from 'lib/cart/actions';
 import { sites } from 'my-sites/controller';
 import config from 'config';
-import CompositeCheckoutContainer from './checkout/composite-checkout-container';
+import { CompositeCheckoutContainer } from './checkout/composite-checkout-container';
```

Comment:
`CompositeCheckoutContainer` is the default export of its file, so I don't think this should be working 🤔